build/build/src/engine/context.rs

```diff
@@ -649,5 +649,17 @@ pub async fn runner_sanity_test(
         output.contains(factorial_expected_output),
         "Native runner output does not contain expected result '{factorial_expected_output}'. Output:\n{output}",
     );
+    if enso_java.is_none() {
```

Comment:
Espresso cannot run `--run test/Base_Tests ^Text` as that requires access to _Truffle RegEx_ and _Truffle RegEx_ doesn't work on _small JDK_ (created by `jlink`) until 
- https://github.com/oracle/graal/issues/8889
 
gets propagated to some release.